### ISSUE TITLE
Vagrant installation not working

### ISSUE THREAD

**Issue by andyjeffries** [flynn/flynn#2874]
I'm following the instructions on https://flynn.io/docs/installation/vagrant and using commit 168108422f6bfc12132d56a9ca9db33b64d4d218 but I get this error when doing `make init`:

```
linux-headers-3.19.0-39-generic is already the newest version.
Suggested packages:
  debhelper
The following packages will be DOWNGRADED:
  dkms
0 upgraded, 0 newly installed, 1 downgraded, 0 to remove and 47 not upgraded.
Need to get 72.4 kB of archives.
After this operation, 1,024 B disk space will be freed.
E: There are problems and -y was used without --force-yes
==> default: Checking for guest additions in VM...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

apt-get install -y linux-headers-`uname -r` dkms

Stdout from the command:

Reading package lists...
Building dependency tree...
Reading state information...
linux-headers-3.19.0-39-generic is already the newest version.
Suggested packages:
  debhelper
The following packages will be DOWNGRADED:
  dkms
0 upgraded, 0 newly installed, 1 downgraded, 0 to remove and 47 not upgraded.
Need to get 72.4 kB of archives.
After this operation, 1,024 B disk space will be freed.


Stderr from the command:

stdin: is not a tty
E: There are problems and -y was used without --force-yes

make[1]: *** [up] Error 1
make: *** [init] Error 2
```

**Comment by titanous** [comment]
I have not been able to replicate this. What versions of Vagrant and the box do you have? What OS? Do you have any Vagrant plugins installed?

**Comment by andyjeffries** [comment]
Vagrant 1.7.4
/Users/andy/.vagrant.d/boxes/flynn-base/20160512.0 (downloaded when I ran it)
Oracle VM VirtualBox Manager 5.0.6
OS X El Capitan
vagrant plugin list:
- vagrant-parallels (1.5.0)
- vagrant-share (1.1.4, system)
- vagrant-vbguest (0.10.0)

**Comment by titanous** [comment]
Try adding this to the Vagrantfile:

```text
config.vbguest.auto_update = false
```

**Comment by andyjeffries** [comment]
Just tried it again (to make sure it's not something screwy that was just a one-off), had the same result.

Tried adding your line and ran it again, and WOW!!!!! First time using Flynn and I LOVE it!

Thanks for your help. I was considering using Deis (believing incorrectly that Flynn was difficult to install), but when they released Deis Workflow forcing people to Kubernetes I decided I should look around. So glad I did!
